src/boot/bootctl-random-seed.c

```diff
@@ -101,6 +135,8 @@ int install_random_seed(const char *esp) {
         if (esp_fd < 0)
                 return log_error_errno(errno, "Failed to open ESP directory '%s': %m", esp);
 
+        (void) random_seed_verify_permissions(esp_fd, S_IFDIR);
```

Comment:
Wouldn't this print "Random seed mount point '/efi/'" or so? Isn't that confusing? Why is the check being done on the esp itself?

Reply:
Almost always the ESP is VFAT, and hence it makes sense to check the top-level inode of it, since it indicates what the whole fs is mounted like, that's all.

And yes, the messages says "Random seed mount point /efi/" which is precisely correct, no, and is what people should fix.

Reply:
I don't know, I think as a user I'd find it confusing if a log message said that /efi/ is the mount point for the random seed, given it's actually for the whole ESP

Reply:
We can certainly reowrd this. "Mount point backing random seed…" sounds better?

Reply:
Yes, that would be better

Reply:
changed

Reply:
> I don't know, I think as a user I'd find it confusing if a log message said that /efi/ is the mount point for the random seed, given it's actually for the whole ESP

/efi/ is the mount point for the random seed though, just like how / is the mount point for /usr/bin/yes and /run is the mountpoint for /run/cups/cups.sock.

Reply:
sure, but files can be over-mounted too, and for a casual user the new message is clearer